### PR TITLE
add containers extra spec

### PR DESCRIPTION
**Pull request by gaelgatelement**

### CODE REVIEW

File: templates/web-deployment.yaml [hunk 1]
```diff
@@ -126,6 +126,9 @@ spec:
       {{- if $.Values.web.extraVolumeMounts }}
         {{- toYaml $.Values.web.extraVolumeMounts | nindent 8 }}
       {{- end }}
+        {{- with $.Values.web.containerExtraSpec }}
+          {{- toYaml . | nindent 10 }}
+        {{- end }}
```
Comment (benbz) [inline templates/web-deployment.yaml:131]:
```suggestion
      {{- with $.Values.web.containerExtraSpec }}
        {{- toYaml . | nindent 8 }}
      {{- end }}
```

File: templates/jvb-statefulset.yaml [hunk 2]
```diff
@@ -66,6 +66,9 @@ spec:
           requests:
             cpu: 100m
             memory: 100Mi
+        {{- with $.Values.jvb.monitoring.containerExtraSpec }}
```
Comment (benbz) [inline templates/jvb-statefulset.yaml:69]: Either this one or the below is likely incorrect as these should be at the same level?

**Review by benbz — COMMENTED** [review]
Not 100% sure sure my comments are correct or that I haven't missed any. I think we need confirmation that all these work via testing

File: templates/jvb-statefulset.yaml (hunk 2 above)
Comment (gaelgatelement) [reply templates/jvb-statefulset.yaml:69]: Indeed there was a mistake, I missed the monitoring spec on my second pass.

File: templates/web-deployment.yaml (hunk 1 above)
Comment (benbz) [reply templates/web-deployment.yaml]: Resolved in error? If not can you say here why this has a different indentation to the `volumeMounts` block above?

**Review by benbz — APPROVED** [review]

**Pull request merged**